<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 71, 105);
border-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="var2_button">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>330</y>
     <width>250</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="var6_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>250</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="var5_button">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>330</y>
     <width>250</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="var3_button">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>480</y>
     <width>250</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="var1_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>250</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="var4_button">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>480</y>
     <width>250</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shuffle_button">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>720</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>720</y>
     <width>91</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>170</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="right_answer_button">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>answer</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="word_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>800</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(200, 211, 223);
font: 17pt &quot;Yrsa&quot;;
color:black;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;
selection-color: rgb(101, 145, 232);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
